src/niscope/metadata/attributes_addon.py

```diff
@@ -160,6 +160,7 @@
     1150078: { "codegen_method": "private" },  # FETCH_OFFSET - Fetch attribute
     1150079: { "codegen_method": "private" },  # FETCH_RECORD_NUMBER - Fetch attribute
     1150080: { "codegen_method": "private" },  # FETCH_NUM_RECORDS - Fetch attribute
+    1150043: { "codegen_method": "private" },  # MEAS_ARRAY_GAIN - Measurement library private - #810
```

Comment:
What about MEASU_ARRAY_OFFSET? That still shows up in RST and references private enum:
```
 +    :py:data:`~niscope._ArrayMeasurement.HANNING_WINDOW`
```

Reply:
There's a whole slew (yes, that is a technical term) of attributes that begin with `MEAS_` which presumably means they apply to the measurement library. All made private

Reply:
We should have an NI-SCOPE expert review what things were made private.